cmd/agent/app/stop.go

```diff
@@ -31,7 +31,7 @@ func init() {
 	AgentCmd.AddCommand(stopCmd)
 }
 
-func stop(*cobra.Command, []string) error {
+func stop(cmd *cobra.Command, args []string) error {
```

Comment:
Why add back the variable ? Having the type with no variable is a common way among golang packages to signal that those parameters aren't used anywhere in the function.

Reply:
leftover from the reverted `kill` command; removing.